.travis.yml

```diff
@@ -0,0 +1,39 @@
+sudo: false  # Use container-based infrastructure
+language: python
+python:
+  - "2.7"
+  - "3.4"
+addons:
+  apt:
+    packages:
+      # Needed for NetMHC
+      tcsh
+env:
+  global:
+    # MHC_BUNDLE_PASS
```

Comment:
It comes from `secure` down below. `MHC_BUNDLE_PASS` (and its value) is encrypted inside the `secure` statement.